### PR TITLE
S2task2

### PR DESCRIPTION
**Pull request by jesshamby**
Added the php and sql to validate user inputs and add new validated inputs to the db

### CODE REVIEW

File: main_page.php [hunk 1]
```diff
@@ -32,14 +32,18 @@
...
-        <label for="recipe">Recipe:</label>
-        <input name="recipe" type="text">
-        <label for="cuisine">Cuisine:</label>
-        <input name="cuisine" type="text">
-        <label for="time">Time (mins):</label>
-        <input name="time" type="number">
-        <label for="link">Link to recipe:</label>
-        <input name="link" type="url">
+        <label for="recipe">Recipe:
+            <input name="recipe" type="text">
```
Comment (Roomsycakes01) [inline main_page.php:36]: slap in id tag on the inputs that matches the for on the labels :)

File: tests/functionsTest.php [hunk 2]
```diff
@@ -30,21 +30,30 @@ function testSuccessCreateRecipeCards2()
...
-        $inputa = "hello";
+        $inputa = 'hello';
 
         $this->expectException(TypeError::class);
 
         $case = createRecipeCards($inputa);
     }
 
-    function testSuccessLinkValidation()
+    function testSuccessValidateString()
```
Comment (Roomsycakes01) [inline tests/functionsTest.php:40]: I think the functions need the public prefix

**Review by Roomsycakes01 — COMMENTED** [review]
Looks good, theres only some minor nitpicks. Good job jess! :)

File: tests/functionsTest.php (hunk 2 above)
Comment (jesshamby) [reply tests/functionsTest.php:40]: I don't think so for unit testing? At least not in my notes - may wait and see if Ash says anything

**Pull request merged**